internal/backends/python/python.go

```diff
@@ -185,65 +196,44 @@ func pythonMakeBackend(name string, python string) api.LanguageBackend {
 			return results
 		},
 		Info: func(name api.PkgName) api.PkgInfo {
-			outputB := util.GetCmdOutput([]string{
-				python, "-c",
-				util.GetResource("/python/pypi-info.py"),
-				string(name),
-			})
-			var output pypiXMLRPCInfo
-			if err := json.Unmarshal(outputB, &output); err != nil {
-				util.Die("PyPI response: %s", err)
-			}
-			info := api.PkgInfo{
-				Name:        output.Name,
-				Description: output.Summary,
-				Version:     output.Version,
-				HomepageURL: output.HomePage,
-				Author: util.AuthorInfo{
-					Name:  output.Author,
-					Email: output.AuthorEmail,
-				}.String(),
-				License: output.License,
+			res, err := http.Get(fmt.Sprintf("https://pypi.org/pypi/%s/json", string(name)))
+
+			if err != nil {
+				util.Die("HTTP Request failed with error: %s", err)
 			}
-			for _, line := range output.ProjectURL {
-				fields := strings.SplitN(line, ", ", 2)
-				if len(fields) != 2 {
-					continue
-				}
 
-				name := fields[0]
-				url := fields[1]
+			defer res.Body.Close()
 
-				matched, err := regexp.MatchString(`(?i)doc`, name)
-				if err != nil {
-					panic(err)
-				}
-				if matched {
-					info.DocumentationURL = url
-					continue
-				}
+			if res.StatusCode != 200 {
+				return api.PkgInfo{}
```

Comment:
is this what we used to do? Silently failing and returning garbage isn't normaly the move.

Reply:
yeah this matches the current behavior (at least for 404s): https://github.com/replit/upm/blob/4caf33404424a0199db79615a64eb02011e8e93b/resources/python/pypi-info.py#L19-L20

Returning an empty PkgInfo struct will result in a message printing "no such package": https://github.com/replit/upm/blob/2e4a14f6d8d596a178dcf3eb75fc21498e869423/internal/cli/cmds.go#L99-L101

Reply:
updated to only return an empty struct on 404s and die otherwise

Reply:
seems to be a standard pattern in the other language backends: https://github.com/replit/upm/blob/a3fb24a65e201bd582303ec0d292ec9f4bf6c0f8/internal/backends/java/java.go#L281-L283